lambdas/initialise-ipv-session/src/main/java/uk/gov/di/ipv/core/initialiseipvsession/InitialiseIpvSessionHandler.java

```diff
@@ -291,7 +303,8 @@ public APIGatewayProxyResponseEvent handleRequest(
         }
     }
 
-    private Optional<InheritedIdentityJwtClaim> getInheritedIdentityClaim(JWTClaimsSet claimsSet)
+    private Optional<ListOfStringValues> getStringListClaim(
+            JWTClaimsSet claimsSet, boolean isForInheritedIdentityClaim)
             throws ParseException, RecoverableJarValidationException {
         if (!configService.enabled(CoreFeatureFlag.INHERITED_IDENTITY)) {
```

Comment:
This check should probably be moved out of this method - it's specific to the inherited id case